src/app/work-item/work-item-board/work-item-board.component.ts

```diff
@@ -90,6 +90,11 @@ export class WorkItemBoardComponent implements OnInit, OnDestroy {
     private areaService: AreaService,
     private filterService: FilterService,
     private route: ActivatedRoute) {
+      router.events.subscribe(event => {
```

Comment:
The challenge is the detail view is a router outlet so we cannot bind any listener.
If you see work-item-board.component.html > <alm-work-item-quick-add
(workItemCreate)="onCreateWorkItem($event)" takes care of updating the lane.